src/app/AlwaysOn.CatalogService/Program.cs

```diff
@@ -41,20 +41,17 @@ public static int Main(string[] args)
 
                 var builtConfig = config.Build();
                 Log.Logger = new LoggerConfiguration()
-                                    .MinimumLevel.Debug()
-                                    .MinimumLevel.Override("Microsoft", LogEventLevel.Information)
+                                    .ReadFrom.Configuration(builtConfig)
                                     .Enrich.FromLogContext()
-                                    .Filter.ByExcluding(c => c.Properties.Any(p => p.Value.ToString().ToLower().Contains("health/liveness"))) // Exclude pod health probe from logging
```

Comment:
No filtering anymore, because the log level does it?

Reply:
yes exactly, this is not needed anymore